crates/bifrost/src/providers/replicated_loglet/network.rs

```diff
@@ -356,13 +356,13 @@ impl From<ReplicatedLogletError> for SequencerStatus {
     }
 }
 
-struct WaitAppendedTask {
+struct WaitCommittedTask {
```

Comment:
Not related, but the name was confusing when I was reviewing the required changes.

Reply:
Karma!